scribble/app/views/comments/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h2>Comment</h2>
+
+<p><%= @comment.text %></p>
+<p>Created At: <%= @comment.created_at %></p>
+<p>Updated At: <%= @comment.updated_at %></p>
```

Comment:
Be sure to take a look at `.strftime`, which will let you format the dates appropriately.